### PR TITLE
Add 'Tested in CODE? in PR template'

### PR DESCRIPTION
**Pull request by TBonnin**
Add 'Tested in CODE? in PR template'

The goal is to help making sure not testing in CODE is a conscious decision, and not an oversight.
It would also indicate to reviewers how much manual testing has been done.

What do you think @guardian/dotcom-platform 

Tested in CODE?
No 😁

### CODE REVIEW

**Review by jfsoul — APPROVED** [review]
I like this idea - I think it will be a red flag if someone hasn't tested something in code that a reviewer thinks they should have.  It also slightly discourages not testing on code (when you know you ought to) because it's public info.

**Comment by DiegoVazquezNanini** [comment]
This will add clarity and help the PR review process.

**Review by DiegoVazquezNanini — APPROVED** [review]

**Review by gtrufitt — APPROVED** [review]
💯

**Review by sndrs — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -13,6 +13,8 @@
 
 ## Screenshots
 
+## Tested in CODE?
+
```
Comment (sndrs) [inline .github/PULL_REQUEST_TEMPLATE.md]: how about 

```
## Tested on CODE?
✅ 
❌
```
so you have to delete something too?

Comment (sndrs) [inline .github/PULL_REQUEST_TEMPLATE.md]: too slow...

Comment (TBonnin) [inline .github/PULL_REQUEST_TEMPLATE.md]: I thought about it, pretty much the same as writing `yes` or `no`

**Pull request merged**